backend/scripts/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Runs blnd-tender tests.
+# All passed parameters go to --test-arguments parameter of the test executable.
+
+# Dependencies:
+# * Postgres server
+# * pg_tmp (http://eradman.com/ephemeralpg/)
+
+# Prerequisites:
+# * Specify your database user name via 'PGUSER' variable; default user name is your OS username.
+#   If you haven't created a database user yet, run
+#   > sudo -u postgres createuser --interactive --pwprompt
+# * If your database user's password is not empty, specify it in .pgpass file
+#   (see https://www.postgresql.org/docs/9.3/libpq-pgpass.html).
+
+set -e -o pipefail
+
+if ! which initdb > /dev/null; then
+    echo -e "Command 'initdb' not found.\nConsider adding '/usr/lib/postgresql/VERSION/bin/' to your PATH."
+    exit 2
+fi
+
+# A temporal server does not live long (60 sec by default), building tests first.
+stack test --no-run-tests
+
+# Running a server in background.
+# Set log level to warning to avoid polluting NOTICE messages in tests.
+conn_str="$(pg_tmp)&options=-c%20client_min_messages\%3DWARNING"
+
+# Executing tests.
+# This thing wraps every argument into double quotes because that's what stack wants :/
+args=( "${@/#/\"}" ); args=( "${args[@]/%/\"}" )
+TEST_PG_CONN_STRING="$conn_str" stack test --no-nix-pure --test-arguments "${args[*]}"
+
+# pg_tmp shutdowns and cleans up automatically after a period of inactivity
```

Comment:
Some extra comments about this file:
1. Is it supposed to be equivalent to what we are doing in `backend/default.nix`? AFAIU it doesn't look equivalent currently, e. g. we pass `-w 600` there and don't pass here. I think it'd be nice to unify as much as possible, though it's minor. `client_min_messages` is another difference.
2. It assumes `stack`, but I prefer `cabal`, so it'd be nice to make it work with `cabal` as well if possible. In `Makefile` we use `EDNA_USE_CABAL` variable to determine what to use. Since you are not using `cabal`, I guess I can try doing it myself. One option is to just call `make` and then expect `edna-test` to be in the `$PATH` and just call it directly. Currently `make` doesn't put it there, but probably it's doable. If we manage to put `edna-test` into `$PATH`, we can avoid weird code that wraps arguments passed to `--test-arguments`.
3. If all this stuff is necessary to make test-suite work, we should update `make test` as well. We can call this script from there or inline it there.

Reply:
1. 600 is default value as I can see from comments. I don't know but maybe `client_min_messages` is different for CI for some reasons (maybe we do not need warnings). What do you think about it and what do you think we need to use?
2. I do not use `cabal` so you are welcome to do it. But if you want I can investigate how to do it by myself